src/classes/OPP_StageMappingCMT.cls

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
For any new files, can you update ApexDocContent?  For the new files for this, I believe they would go in the Opportunity.htm file, in the Stage to State Section at the bottom.

Reply:
Added OPP_StageMappingCMT to the Opportunity.htm file